### ISSUE TITLE
what missiles fit in VLS cell

### ISSUE THREAD

**Issue by lordi-erez** [BrettRyland/BDArmory-Extended#26]
i try to attach missiels to top node, they don't stick

**Comment by 117StinkBug** [comment]
The VLS cell is adjustable, enabling it to fit a variety of munitions. 

As for the missiles not sticking, the sound of the missile disappearing is expected as the part works by getting rid of the part you placed and spawning the desired missile in the cell. you will need to adjust the cell accordingly as the default values won't work for larger missiles. the missile(s) loaded won't count towards the part count like they do with some of the other launchers.

**Comment by lordi-erez** [comment]
i meant that the missile doesnt even disappear, it doesnt attach to the node

**Comment by 117StinkBug** [comment]
Perhaps it is the orientation. Have you tried the WASDQE keys to rotate the missile?

**Comment by lordi-erez** [comment]
yes, i have tried, however 2 missiles are confirmed to successfully connect (as of the time i type this comment):

DAGR rockets
RIM-66D

and thats it, nothing else connects to node, no mater what i try
